src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/AbstractMicrosoftTeamsSectionElement.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ */
+abstract class AbstractMicrosoftTeamsSectionElement implements MicrosoftTeamsSectionElementInterface
```

Comment:
Is this really needed? Only use twice..

Reply:
The toArray() method is defined in the interface and implemented in the abstract one, across all type of options, seems ok to avoid the duplication of the to Array method